src/Repository/IssueRepository.php

```diff
@@ -78,6 +78,7 @@ public function getClosedIssuesFromInterval(Project $project, \DateTimeInterface
     private function getClosedStatuses(Project $project): array
     {
         // @TODO Get this from project somehow.
+        // SELECT * FROM zp_settings WHERE `key` = 'projectsettings.«project id».ticketlabels'
```

Comment:
Øhh... remove or elaborate on why this comment should be added.

Reply:
It's an important note to (my)self. Now it's has been removed.